src/Slider.jsx

```diff
@@ -29,7 +29,7 @@ function killEvent(ev) {
 
 class Button extends React.Component {
   render() {
-    return <button {...this.props} type="button" />;
+    return <button {...this.props} aria-label="Slider Handle" type="button" />;
```

Comment:
It's a balance between a clean API and a convenient one - having two props is more convenient for those who only want to override the label, but having one prop is a cleaner API that avoids bikeshedding about any future modifications we'd want to make easier.